src/Avalonia.Controls/Button.cs

```diff
@@ -25,7 +22,7 @@ public enum ClickMode
         /// <summary>
         /// The <see cref="Button.Click"/> event is raised when the pointer is pressed.
         /// </summary>
-        Press,
+        Press
```

Comment:
Nit: unneeded change. These things aren't super-important but they do mess with `git blame`. If you can be bothered you can go back and undo changes like this [using git rebase](https://stackoverflow.com/a/1186549/6448).